cmd/minikube/cmd/dashboard.go

```diff
@@ -33,7 +33,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
-	configcmd "k8s.io/minikube/cmd/minikube/cmd/config"
+	pkgaddons "k8s.io/minikube/pkg/addons"
```

Comment:
Can this be made just "addons"?

Reply:
so the compiler complains that there's a constant named [addons](https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/start.go#L106) in the same package :(